test/unit/presenters/publishing_api/html_attachment_presenter_test.rb

```diff
@@ -74,6 +68,7 @@ def present(record)
     assert_equal "cy", present(html_attachment).content[:locale]
   end
 
+  # Is this really useful?
```

Comment:
I think we discussed this briefly, that we'd expect it to be covered by some kind of validation rule but we should check.

Reply:
💯 

I did have a look into this, and I think we're safe to remove it thanks to validation rules.

See the final commit on this PR: 798eae08cee75f4249c5847359a293b59d4b6290

Reply:
Aha! Yep just got to it!